src/main/java/com/endava/soa/pageObject/AdaugaAnuntPage.java

```diff
@@ -0,0 +1,9 @@
+package com.endava.soa.pageObject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AdaugaAnuntPage {
```

Comment:
Please use English name convention both for class name and fields name.

Reply:
this is outdated